b2b-sample/guardio-insurance-sample-application/README.md

```diff
@@ -17,3 +17,9 @@ yarn install
 yarn dev
 ```
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+## Additionally
+* If the correct organisation id and organisation name are returned in the idToken when a user is logging into an organisation, remove the `SAMPLE ORGS` section from the <i>config.json</i> file.
```

Comment:
Suggestion

```
"If the correct organization ID and organization name are returned in the IdToken when a user logs into an organization, remove the SAMPLE_ORGS section from the <i>config.json</i> file.
```

Reply:
These have been fixed with the new README.md file

### File
- https://github.com/wso2/samples-is/tree/master/b2b-sample#readme